plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestingSqlServer.java

```diff
@@ -30,6 +34,7 @@
     private final MSSQLServerContainer<?> container;
     private final boolean snapshotIsolationEnabled;
     private final String databaseName;
+    private Closeable reusableContainer = () -> {};
```

Comment:
as part of unification we should remove the `start` method and initialize in ctor (as we do in others)

followup perhaps

Reply:
This is something I don't like. Starting resources in constructor has short legs. Notice if you start a container in constructor and then exception is thrown there is no option to call `close()`. Maybe it is not a big deal, as we have failure anyway, but dangling container could influence other tests.

Reply:
This is a design pattern we apply consistently except here. I am OK changing the principle, but as long as we did not change it, I do not see a compelling reason for TestingSqlServer to be different from others API-wise

Reply:
Sounds good.